lbrynet/daemon/Daemon.py

```diff
@@ -939,6 +951,33 @@ def _get_lbry_files(self, return_json=False, full_status=True, **kwargs):
         log.debug("Collected %i lbry files", len(lbry_files))
         defer.returnValue(lbry_files)
 
+    def _sort_lbry_files(self, lbry_files, sorts):
+        for field, order in sorts:
+            reverse = order == FILE_SORT_ORDER_DESCENDING
```

Comment:
Would you consider `direction` instead of `order` and `is_reversed` instead of `reverse`?